repository-data/webfiles/src/main/resources/site/freemarker/hee/macros/quick-links.ftl

```diff
@@ -9,9 +9,9 @@
         <div class="nhsuk-grid-column-one-third">
             <div class="nhsuk-card">
                 <div class="nhsuk-card__content">
-                    <h3 class="nhsuk-card__heading"><@fmt.message key="quicklinks.header"/></h3>
```

Comment:
@thojava - Could we get rid of `<@hst.setBundle basename="uk.nhs.hee.web.global"/>` ?

Reply:
Done

Reply:
Thank you @thojava!